Add nginx route relation doc updates (#103)
canonical/nginx-ingress-integrator-operator: docs/how-to/add-the-nginx-route-relation.md

```

> This will build the charm inside an LXC container for you. The output will tell you the location of the built charm. For example, `my-charm_ubuntu-22.04-amd64.charm`.

Reviewer: Nitpicky, but I think the tone of the documentation here might be a little to friendly.
```suggestion
This will build the charm inside an LXC container. The output is the location of the built charm. For example, `my-charm_ubuntu-22.04-amd64.charm`.
```

> Next, add a juju model and deploy the charm.

Reviewer: ```suggestion
Add a juju model and deploy the charm.
```

```
juju add-model ingress-test
juju deploy ./my-charm_ubuntu-22.04-amd64.charm --resource httpbin-image=kennethreitz/httpbin
```

> To inspect the deployment, run `juju status`. Once the application reaches a status of `active idle` our application has been deployed. We can visit it in a browser by getting the IP address of the unit and then going to `http://${ip_of_unit}`.

Reviewer: ```suggestion
To inspect the deployment, run `juju status`. Once the application reaches a status of `active idle` the application has been deployed. Visit it in a browser by getting the IP address of the unit and then going to `http://${ip_of_unit}`.
```
Perhaps an example of the output of `juju status` would be nice to direct the user into where the unit-ip is in the output :D

> So we now have a working charm, great! However, what we don't currently have is ingress for our application configured. MicroK8s sets up networking in such a way that you can reach the IPs of units directly, but in a production Kubernetes cluster things don't work in this way. Also, we're visiting a cluster-internal IP address directly, what if we want a real hostname/IP address for this? We'll need to configure Ingress for this to be possible.

Reviewer: ```suggestion
The steps thus far don't include ingress for the application. MicroK8s sets up networking in a way that unit IPs can be reached directly, but in a production Kubernetes cluster this is not the case. To allow real hostnames/IP addresses, configure Ingress.
```
